tests/queries/0_stateless/02884_interval_operator_support_plural_literal.sql

```diff
@@ -0,0 +1,16 @@
+SELECT INTERVAL 2 year;
```

Comment:
Does it work the same way with `INTERVAL '2 seconds'`? Could you add a test for that?